### DISCUSSION TITLE
Fan speed and on/standby values?

### DISCUSSION THREAD

**Discussion by StefanR71** [raomin/ESPAltherma#223] [Q&A]
Hello,

great project, thank you very much for that. It's working very well, but i have now two questions.

My devices are:

Inside: ETBX16EF9W
Outside: EPRA18DAW1

I currently use the ALHERMA(TOP-GRADE) file, if there is a better (optimal) file for that devices, please give me a hint.

My first question is now regardig the fan speed value, is that a percent value? Or is this RPM wich must be calculated to a correct value? Or a value between 0 and X, but what is X for the highest value?

And the second question, is there a value regarding the state of the cooling/heating and the hot water part?

From the LAN adapter i can query two values regarding the heating/cooling:
1. on or standby
2. heating, cooling or auto

For the hot water is only one value:
1. on or standby

Has someone seen that state in ESPAltherma too?

Kind regards
Stefan

**Comment by bzq-pl** [comment]
Hi Stefan,

I can't help you in your questions - since I'm also brand new in using ESPAltherma but could you be so kind and share more details about reading from LAN adapter. How do you this?

Regards
Piotr

**Comment by StefanR71** [comment]
Hi Piotr,

i made my own python script from this: https://community.openhab.org/t/how-to-integrate-daikin-altherma-lt-heat-pump/16488/14

And here are some additional infos: https://github.com/SmartHomeIntegrationTech/MQTTServices-Daikin

But here we get less temperature etc. values, so for monitoring the ESPAltherma give much more detailed informations.

On the other side, with:

setValue(ws, "1/Operation/OperationMode", "\"heating\"")
or:
setValue(ws, "1/Operation/Power", "\"standby\"")

i'm able to change the modes from standby to on or change from heating to cooling or to auto.

Because this i'm using both.

Official the LAN adapter BRP069A61 isn't supported on my model, but for query and set vaulues it still works. My model has a built in WLAN, but here i didn't find a way to query and set values, this is only to communicate with the cloud to use the APP.

Regards
Stefan

**Reply by bzq-pl** [reply]
thx! so your inside module doesn't have ethernet interface and you use BRP069A62 LAN adapter. My ground-heat pomp EGSAH-D9W has network interface and I was wondering if I can get some info from this interface. Nevertheless ESPAltherma project is still in my agenda since this solution is read-only and should be great for monitoring.

Regards
Piotr

**Comment by StefanR71** [comment]
Oh, just found in the WiKi informations about values for the fan a description:  (10 rpm) means that you should multiply by 10 to get rpm value. eg 5 -> 50rpm

So only the operating modes i'm still searching...

**Comment by raomin** [comment]
Hi @StefanR71 
Yes `TOPGRADE` is probably what is best for your High temp HP.
Have you looked at:
```
//{0x62,2,306,1,-1,"Storage eco ON/OFF"},
//{0x62,2,305,1,-1,"Storage comfort ON/OFF"},
//{0x62,2,304,1,-1,"Powerful DHW Operation. ON/OFF"},
//{0x62,2,303,1,-1,"Space heating Operation ON/OFF"},
....
//{0x62,7,305,1,-1,"Main RT Cooling"},
//{0x62,7,304,1,-1,"Main RT Heating"},
```

You can also give a try to `ALTHERMA(TOP-GRADE-SMALL_E)TEST.h`

**Comment by StefanR71** [comment]
Thank you for the answer, yes:

//{0x62,2,303,1,-1,"Space heating Operation ON/OFF"},

is set to OFF when "heating/cooling" is set to standby, this one i can use, but:

//{0x62,7,305,1,-1,"Main RT Cooling"},
//{0x62,7,304,1,-1,"Main RT Heating"},

Is every time set to OFF, even if the device is working...

The same for:

//{0x62,2,304,1,-1,"Powerful DHW Operation. ON/OFF"},

Is every time OFF, because this is a option in the menu which enable a faster powerful hot water function. The last two values i havn't queryed yet, but this are also additional hot water functions which can be scheduled within a daily program:

//{0x62,2,306,1,-1,"Storage eco ON/OFF"},
//{0x62,2,305,1,-1,"Storage comfort ON/OFF"},

So i think this willl not show the needed values.

What i'm missing:

1. A general DHW on/off state.
2. The operating mode: only "heating", only "cooling" or "auto"

This value:

//{0x10,0,217,1,-1,"Operation Mode"},

Perhaps this show only "Heating" when he is heating the room and makes hot water and "Cooling" (not seen yet) when he is in cooling the room. This seems to be a general operating mode what he is doing and not what is configured.

Then we have the:

//{0x60,2,315,1,-1,"I/U operation mode"},

This is strange, when he is heating the room (DHW enabled), the value reports "Heating". When he is making hot water the value reports "Heating + DHW". When i disable the room heating the value reports only "DHW". Why is he showing here "Heating + DHW"?

Perhaps this table makes it more clear:

heating on | DHW on | current operating heating | value shows "Heating"
heating on | DHW on | current operating DHW    | value shows "Heating + DHW"
heating off | DHW on | current operating DHW    | value shows "DHW"
heating off | DHW on | current operating nothing | value shows "DHW"
heating on | DHW on | current operating nothing | value shows "Heating"

The "Heating + DHW" don't make sense to me, but i think this value will be delivered by the device, correct?

I think at the moment the best is to query this with my python script, i must still use the LAN adapter to adjust the temperatures and switch on/off the the DHW or room heating.

Regards
Stefan

**Reply by raomin** [reply]
My understanding (might be wrong...) is the same as yours
`0x10` registry is for outdoor unit: It heats/cools/or do nothing.
`0x60` is the internal unit, in our case, what is it using the heat produced for...
- if it uses heat for heating DHW only, it shows `DHW`
- if it uses heat for heating DHW and the house, it shows `Heating + DHW`
- if it uses heat for heating only the house (when the DHW does not need to be heated), it shows `Heating`
- When it does nothing it shows (wrongly IMO) `DHW` and you have to check another parameter, eg. the `{0x10,0,217,1,-1,"Operation Mode"}` to know if it does nothing or if it heats water.

**Reply by StefanR71** [reply]
The {0x10,0,217,1,-1,"Operation Mode"} shows every time "heating".

For my understanding the "Heating + DHW" is wrong, because the device can't do both at the same time. If it shows the current configuration it must show "Heating + DHW" when he is currently heating the house and DHW is enabled. But in my case he shows only "Heating". He shows only "Heating + DHW" when in DHW operation.

If he is doing DHW, this i can see at the state of the 3 way valve.

**Comment by StefanR71** [comment]
Hi,

i must ask here again, because i can't query the state with the python script and the Daikin LAN adapter, it stopped working from one day to the other. But i still can control the unit, so setting the DHW temperature, switching on and off is still working.

Because this i'm searching again a way  to query the current configuration state...

This i'm searching:
DHW is in general on or off? So not what he is currently doing, i neet to know if this is switched on or off.

The same for heating/cooling. Here i'm searching the info if this is in genereral switched on or off and the operating mode, if he is currently setup for heating, cooling or in automatic mode for heating or cooling.

Has here someone foud that informations?

**Comment by StefanR71** [comment]
Okay, here is still the {0x62,2,303,1,-1,"Space heating Operation ON/OFF"} the only value i can use, all other states it seems that we can't get them. I have enabled all possible values even the values without a description and made a compare before and after changing the state. No changes...

We have now ordered the Daikin HomeHub - EKRHH, this has a ModBus RTU and TCP interface. On the RTU interface i will connect also a ESP32 with a RS485 to TTL converter and the relay board to realize my relay problem and all the other states i can query via ModBus TCP, i hope. :-)
